recipe/meta.yaml

```diff
@@ -73,10 +73,12 @@ test:
     - jinja2
     - cffi
     - scipy
+    # At least NumPy 1.26.0 has an issue with arctanh on glibc<=2.17.
+    # ref: https://github.com/conda-forge/numba-feedstock/pull/128#issuecomment-1795031396
+    - numpy !=1.26.0  # [linux and x86_64]
     - ipython
     - setuptools
     - tbb
-    - llvm-openmp              # [osx]
     # Need these for AOT. Do not init msvc as it may not be present
     - {{ compiler('c') }}      # [not win]
     - {{ compiler('cxx') }}    # [not win]
```

Comment:
Somewhat unrelated, but do we need to limit this on Windows? We should always have MSVC on CI. Or do we want to keep this around for offline building/testing?

Reply:
I guess we could remove it.
It comes from the upstream recipe.
IDK in which way it changes the tests, TBH.

Reply:
No strong feelings. Agree dropping altogether seems reasonable. We can always change later if needed

Reply:
Ah, I actually meant to drop the selector only but deleted the lines :facepalm:.

Reply:
Oh ok. Thought it was dropped since [the current AOT implementation is deprecated]( https://numba.readthedocs.io/en/0.58.1/reference/deprecation.html#deprecation-of-the-numba-pycc-module )

Though ok with continuing to test AOT until it is removed